lib/logdna.rb

```diff
@@ -60,32 +60,26 @@ def level=(value)
     end
 
     def log(message = nil, opts = {})
-      if message.nil? && block_given?
-        message = yield
-      end
+      message = yield if message.nil? && block_given?
```

Comment:
the option that changes the if statement is actually turned off in my PR. The previous version is preferable

Reply:
That's automatically fixed and autocorrected by `rubocop` - from now on, `Rubocop` is gonna serve here as a linter and syntax checker, @vilyapilya

Reply:
I think it'd be better to have standard if statements. We can configure rubocop to our own needs.

Reply:
I don't think this change adds to readability